src/config/texts.js

```diff
@@ -41,6 +41,14 @@ export const texts = {
     operatingCompany: 'Anbieter',
     prices: 'Preise'
   },
+  pushNotifications: {
+    approve: 'Jetzt einschalten',
+    decline: 'Vielleicht später',
+    greetingBody: 'Damit dich wichtige Mitteilungen erreichen, würden wir dir gerne Benachrichtigungen schicken können. Du kannst diese Einstellung jederzeit in deinem persönlichen Bereich ändern.',
```

Comment:
do you copied this from another app or defined it by yourself?
i always try to avoid "du" or "sie" with more generic sentences. maybe we need to consult @pwilimzig to find the right solution that works for all customers.

Reply:
we will receive texts soon by him.

Reply:
updated with the preliminary text we received from him

will resolve this for now.